R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
Hmmmm. I am sure if GH would be happy about this. It is just a way going around (well, loosen) their rate limits. Also, I don't think we could implement realms without requiring authentication.

I think a token is fine for now, but maybe I would create a `devtools-robot` user for it.

Btw. what is GH's approach to publishing these tokens? Is that OK?